src/resources/Incorporations/limitedCompany.ts

```diff
@@ -47,36 +47,23 @@ export const LimitedCompanyResource: IncorporationResourceIF = {
   incorporationAgreement: {
     helpSection: [
       {
-        header: 'What is the sample Incorporation Agreement and Table 1 Articles?',
+        header: 'What is the sample Incorporation Agreement and BC Limited Company Articles?',
         helpText: [
-          `The sample Incorporation Agreement and Table 1 Articles is a template that you can use to create an
-            incorporation agreement and articles for your company. It uses all the standard provisions suggested by
-            legislation.`,
-          `If you would like to customize any provisions in the Articles, you cannot use this sample. We recommend
-            seeking professional assistance from a lawyer or accountant to help you prepare your Articles.`
+          `The sample Incorporation Agreement and Company Articles is a template that you can use to create an 
+            incorporation agreement and articles for your company. It uses all the standard provisions by legislation. 
+            There are three types of samples depending on if you’re incorporating a Limited Company, a Benefit Company, 
+            or a Community Contribution Company. There is currently no Incorporation Sample for Unlimited 
+            Liability Companies.`,
+          `If you would like to customize any other provisions in the Articles, you cannot use these samples. 
+            We recommend seeking professional assistance from a lawyer or accountant to help you prepare your articles.`
         ]
       },
       {
-        header: 'Can I use the sample Incorporation Agreement and Table 1 Articles?'
-      },
-      {
-        header: 'You can use the Incorporation Agreement and Table 1 Articles if:',
-        icon: 'mdi-check',
-        iconColor: 'green darken-2',
-        statements: [
-          `There are no special rights or restrictions attached to any class or series of shares in
-            the corporation's authorized share structure.`,
-          'You do not wish to change any of the standard provisions in the sample Articles.'
-        ]
-      },
-      {
-        header: 'You cannot use the Incorporation Agreement and Table 1 Articles if:',
-        icon: 'mdi-close',
-        iconColor: 'red',
-        statements: [
-          `There are special rights or restrictions attached to any class or series of shares in the corporation's
-            authorized share structure.`,
-          'You wish to change any of the standard provisions in the sample Articles.'
+        header: 'What is a Benefit Provision?',
+        helpText: [
+          `A Benefit Provision is a statement by the company of its public benefits and its commitments to promote those
+            public benefits and to conduct business in a responsible and sustainable manner.`,
+          `A Benefit Company must include a benefit provision in its Articles.`
```

Comment:
Why is this help here for a BC Limited Company?

Reply:
Again, grabbed straight from the ticket. I can confirm with Yui.

Reply:
Confirming with Ethan/ui team.

Reply:
Any update on this?